source/developers-guide/plugin-quick-start/index.md

```diff
@@ -525,3 +595,520 @@ Possible settings for the parameter `caches` contains each context as constant.
+{
+    public static function getSubscribedEvents()
+    {
+        return [
+            'Enlight_Controller_Front_RouteStartup' => 'onRouteStartup'
+        ];
+    }
+
+    public function onRouteStartup(\Enlight_Controller_EventArgs $args)
+    {
+        $sloganPrinter = $this->container->get('swag_slogan_of_the_day.slogan_printer');
+        $sloganPrinter->print();
+    }
+}
+```
+
+`SwagSloganOfTheDay/Resources/services.xml`
+
+```xml
+<?xml version="1.0" ?>
+
+<container xmlns="http://symfony.com/schema/dic/services"
+           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+
+    <services>
+        <service id="swag_slogan_of_the_day.slogan_printer" class="SwagSloganOfTheDay\SloganPrinter">
+            <argument type="service" id="dbal_connection" />
+        </service>
+    </services>
+</container>
+```
+
+### Decorate a service
+The following example shows you how to decorate a service which implements an interface and gets defined in the Shopware dependency injection container.
+```php
+<?php
+
+namespace SwagExample\Bundle\StoreFrontBundle;
+
+use Shopware\Bundle\StoreFrontBundle\Service\ListProductServiceInterface;
+use Shopware\Bundle\StoreFrontBundle\Struct\ProductContextInterface;
+
+class ListProductService implements ListProductServiceInterface
+{
+    private $service;
+
+    public function __construct(ListProductServiceInterface $service)
+    {
+        $this->service = $service;
+    }
+
+    public function getList(array $numbers, ProductContextInterface $context)
+    {
+        $products = $this->service->getList($numbers, $context);
+        //...
+        return $products;
+    }
+
+    public function get($number, ProductContextInterface $context)
+    {
+        return array_shift($this->getList([$number], $context));
+    }
+}
+```
+The original `\Shopware\Bundle\StoreFrontBundle\Service\Core\ListProductService` defined with the service id `shopware_storefront.list_product_service`. The following service definition decorates this service using the service above:
+
+```xml
+<?xml version="1.0" ?>
+
+<container xmlns="http://symfony.com/schema/dic/services"
+           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+
+    <services>
+        <service id="swag_example.list_product_service"
+                 class="SwagExample\Bundle\StoreFrontBundle\ListProductService"
+                 decorates="shopware_storefront.list_product_service"
+                 public="false">
+
+            <argument type="service" id="swag_example.list_product_service.inner"/>
+        </service>
+    </services>
+</container>
+
+```
+
+For more information see http://symfony.com/doc/current/service_container/service_decoration.html
```

Comment:
please link to the symfony 2.8 doc, as this is the version we use

Reply:
d